src/services/helpers/gantt/ganttCell/GanttCell.tsx

```diff
@@ -269,4 +270,24 @@ export function groupAssignments(
   return groupedAssignments;
 }
 
+// this is to display the end confidence as a gantt cell next to the project start gantt cell
+export function getRolesAsRow(role: Role): Role[] {
+  const clonedRole = { ...role };
+  const result = [role];
+  if (
+    clonedRole.endDate &&
+    clonedRole.endConfidence &&
+    clonedRole.endConfidence < 1
+  ) {
+    clonedRole.startDate = clonedRole.endDate;
+    clonedRole.endDate = null;
+    if (clonedRole.endConfidence) {
```

Comment:
I don't think you need this conditional, the conditional its in already checks that `clonedRole.endConfidence` is there
```ts
if (
    clonedRole.endDate &&
    clonedRole.endConfidence &&
    clonedRole.endConfidence < 1
  )
```